examples/game/breakout.rs

```diff
@@ -82,9 +84,15 @@ struct Velocity(Vec2);
 #[derive(Component)]
 struct Collider;
 
+#[derive(Default)]
+struct CollisionEvent;
+
 #[derive(Component)]
 struct Brick;
 
+#[derive(Component)]
```

Comment:
Why is this `Component`?

Reply:
I think it was just a copy paste mistake. It's not needed, I'll remove it as soon as I have access to a computer.